project_euler/problem_024/sol2.py

```diff
@@ -0,0 +1,42 @@
+"""A permutation is an ordered arrangement of objects.
+For example, 3124 is one possible permutation of the digits 1, 2, 3 and 4.
+If all of the permutations are listed numerically or alphabetically
+, we call it lexicographic order.
```

Comment:
better put the comma to the previous line

Reply:
okay !